README.md

```diff
@@ -0,0 +1,75 @@
+# Nuxt 3 Minimal Starter
```

Comment:
[Minor]
Usually custom README.md file is created for each project.

Though it's good to have a least default README generated by the framework.